corehq/apps/userreports/tasks.py

```diff
@@ -367,16 +368,25 @@ def _save_document_helper(indicator, doc):
     queue=settings.CELERY_PERIODIC_QUEUE,
 )
 def async_indicators_metrics():
+    now = datetime.utcnow()
     oldest_indicator = AsyncIndicator.objects.order_by('date_queued').first()
     if oldest_indicator and oldest_indicator.date_queued:
-        lag = (datetime.utcnow() - oldest_indicator.date_queued).total_seconds()
+        lag = (now - oldest_indicator.date_queued).total_seconds()
         datadog_gauge('commcare.async_indicator.oldest_queued_indicator', lag)
 
-    indicator = AsyncIndicator.objects.first()
-    if indicator:
-        lag = (datetime.utcnow() - indicator.date_created).total_seconds()
+    oldest_100_indicators = AsyncIndicator.objects.all()[:100]
+    oldest_indicator = oldest_100_indicators[0]
```

Comment:
This is causing this error https://sentry.io/dimagi/commcarehq/issues/416664542/. PRing this fix which I think is all we want: https://github.com/dimagi/commcare-hq/pull/18969.